api/python/arrus/utils/imaging.py

```diff
@@ -3490,7 +3490,7 @@ def get_min_max_x_y(aperture):
         self.min_tang = self.num_pkg.float32(self.min_tang)
         self.max_tang = self.num_pkg.float32(self.max_tang)
         burst_factor = ref_tx.excitation.n_periods / (2 * self.fn)
-        self.initial_delay = -start_sample / 65e6 + burst_factor + tx_center_delay
+        self.initial_delay = -start_sample / 120e6 + burst_factor + tx_center_delay
```

Comment:
Sampling frequency should depend on AFE(OEM) variant

Reply:
Fixed